tests/test_embedding_timestamps.py

```diff
@@ -41,11 +41,11 @@ def test_timestamps_no_channels(self):
 
         for params in paramlist:
             emb0, ts0 = torchopenl3.get_audio_embedding(
-                sounds, 48000, batch_size=32, **params
+                sounds, 48000, batch_size=32, sampler="resampy", **params
```

Comment:
Why don't we just make "resampy" the default in get_audio_embedding, and make a note about why it's the default?

Reply:
yes we can do that. its just that julius is based on pytorch, so kept this as an option.

Reply:
No if Julius gives bad results then we don't want to include it in this release

Reply:
sure. then, will change that the preprocessing part to numpy. I think that will be fine.